.github/workflows/test-coverage.yml

```diff
@@ -274,6 +274,30 @@ jobs:
           coverage-artifact-name: "coverage_telemetry"
           coverage-path: coverage.txt
 
+  # This job runs the JS tests to ensure they are able to build and run.
+  test-js:
+    name: Test JS job
+
+    runs-on: ubuntu-latest
+
+    env:
+      CGO_ENABLED: 0
+
+    steps:
+      - name: Checkout code into the directory
+        uses: actions/checkout@v4
+
+      - name: Setup defradb
+        uses: ./.github/composites/setup-defradb
+
+      # Disable AppArmor for Ubuntu 23.10+.
+      # https://chromium.googlesource.com/chromium/src/+/main/docs/security/apparmor-userns-restrictions.md
+      - name: Disable AppArmor
+        run: echo 0 | sudo tee /proc/sys/kernel/apparmor_restrict_unprivileged_userns
+
+      - name: Test JS
+        run: make test:js
```

Comment:
todo: Just like the change detector, I don't think this belongs in this file. Main reasons being that this is not generating test coverages.

Reply:
I moved it to a new workflow file.